src/Microsoft.AspNetCore.Identity/UserManager.cs

```diff
@@ -1954,6 +1969,93 @@ public virtual Task<IList<TUser>> GetUsersInRoleAsync(string roleName)
         }
 
         /// <summary>
+        /// Returns an authentication token for a user.
+        /// </summary>
+        /// <param name="user"></param>
+        /// <param name="loginProvider">The authentication scheme for the provider the token is associated with.</param>
+        /// <param name="tokenName">The name of the token.</param>
+        /// <returns></returns>
+        public virtual Task<string> GetAuthenticationTokenAsync(TUser user, string loginProvider, string tokenName)
```

Comment:
keeping tokenName for these since we do the same for the other similar APIs, userName, roleName as opposed to only saying just name